src/components/map/Map.tsx

```diff
@@ -190,6 +195,11 @@ export const Map = (props: MapProps) => {
     ],
   );
 
+  const externalRealtimeMapUrl = getTextForLanguage(
+    configurableLinks?.externalRealtimeMap,
+    language,
+  );
```

Comment:
Also possible to encapsulate this into the `ExternalMapButton` component, by making it `return null` if no url in `configurableLinks`?

Reply:
Sounds like a good change. I've updated to do so now